File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT0101U02GetInsuranceCode.java
package nta.med.data.model.ihis.nuro;

import java.io.Serializable;

public class NuroOUT0101U02GetInsuranceCode implements Serializable{
	private String insuranceCode;

	public NuroOUT0101U02GetInsuranceCode(String insuranceCode) {
		super();
		this.insuranceCode = insuranceCode;
	}

	public String getInsuranceCode() {
		return insuranceCode;
	}

	public void setInsuranceCode(String insuranceCode) {
		this.insuranceCode = insuranceCode;
	}
	
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT0101U02GetInsuranceCode.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable ]
	public class NuroOUT0101U02GetInsuranceCode
	{
		private String _insuranceCode;

		public String InsuranceCode
		{
			get { return this._insuranceCode; }
			set { this._insuranceCode = value; }
		}

		public NuroOUT0101U02GetInsuranceCode() { }

		public NuroOUT0101U02GetInsuranceCode(String insuranceCode)
		{
			this._insuranceCode = insuranceCode;
		}

	}
}